faq.html

```diff
@@ -150,13 +150,15 @@ <h3 class="question">
                 </div>
               </a>
               <div class="faq__text">
-                Learning Git is highly recommended, but if you do not know it
-                yet, please follow
-                <a
-                  href="https://github.com/Real-Dev-Squad/website-welcome/blob/main/CONTRIBUTING.md"
-                  >this
-                </a>
-                guide to get started with Git.
+                <p>
+                  Learning Git is highly recommended, but if you do not know it
+                  yet, please follow
+                  <a
+                    href="https://github.com/Real-Dev-Squad/website-welcome/blob/main/CONTRIBUTING.md"
+                    >this
```

Comment:
Could we please add target attribute so that this page would open in a new tab? https://stackoverflow.com/questions/17711146/how-to-open-link-in-a-new-tab-in-html

Since it's a third party link, better to open it in new tab

Reply:
I will add the target attribute. I will create a new PR for the target attribute.

Reply:
I think we can add target attribute in this PR only as it is a very small fix @kamara-moses